p2p/p2p.go

```diff
@@ -19,77 +19,96 @@ package p2p
 import (
 	"bufio"
 	"context"
-	"fmt"
+	"time"
 
-	module "github.com/ChainSafe/gossamer/internal/api/modules"
-	"github.com/ChainSafe/gossamer/internal/services"
+	"github.com/ChainSafe/gossamer/common"
 	log "github.com/ChainSafe/log15"
 
+	"github.com/libp2p/go-libp2p-core/network"
 	net "github.com/libp2p/go-libp2p-core/network"
 )
 
-var _ services.Service = &Service{}
+// var _ module.P2pApi = &Service{}
+// var _ services.Service = &Service{}
```

Comment:
You can remove the first. The second is optional but should probably stay to ensure we don't change the service's compatibility with the interface.

If you're not familiar with these: they just assert that you can create an instance of `Service` and store it as a `services.Service` (ie. it adheres to the interface). It never actually gets saved due to the `_`, but the compiler will complain if its not possible.

Reply:
Thanks for the explanation! Deleted first and uncommented second.